git_intro/git_intro.md

```diff
@@ -67,6 +66,8 @@ Microsoft Word's [Track Changes](https://support.office.com/en-us/article/Track-
 Google Docs' [version history](https://support.google.com/docs/answer/190843?hl=en), or
 LibreOffice's [Recording and Displaying Changes](https://help.libreoffice.org/Common/Recording_and_Displaying_Changes).
 
+A formal version control system like git extends and formalizes that functionality, giving you a lot more control over your editing history. 
```

Comment:
I added this line to try to draw a clearer path from the above list of tools (track changes, etc.) and the care box that follows --- I was worried it read like those tools were what we were talking about when we say "version control", and they're not quite.